mcs/class/corlib/System.IO/DriveInfo.cs

```diff
@@ -169,14 +169,39 @@ public override string ToString ()
 		}
 
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
-		extern static bool GetDiskFreeSpaceInternal (string pathName, out ulong freeBytesAvail,
+		unsafe extern internal static bool GetDiskFreeSpaceInternal (char *pathName, int pathName_length, out ulong freeBytesAvail,
 							     out ulong totalNumberOfBytes, out ulong totalNumberOfFreeBytes,
 							     out MonoIOError error);
 
+		unsafe static bool GetDiskFreeSpaceInternal (string pathName, out ulong freeBytesAvail,
```

Comment:
Why isn't that one `internal` anymore? If it doesn't need to be then that's actually better, otherwise you are just breaking the API.

Reply:
I didn’t intend to change existing access. Will double check.

Reply:
There is no access change here, just adding unsafe.